sagemaker-clarify/online_explainability/tabular_multi_model_endpoint/multi_model_xgboost_with_online_explainability.ipynb

```diff
@@ -0,0 +1,1823 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ddf1b419",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# SageMaker Clarify Online Explainability on Multi-Model Endpoint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "885111e2",
+   "metadata": {},
+   "source": [
+    "## Runtime\n",
+    "This notebook takes approximately 20 minutes to run"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e01c746e",
+   "metadata": {},
+   "source": [
+    "## Contents"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37b23936",
+   "metadata": {},
+   "source": [
+    "* [Introduction](#Introduction)\n",
+    "* [General Setup](#General-Setup)\n",
+    "  * [Install dependencies](#Install-dependencies)\n",
+    "  * [Import libraries](#Import-libraries)\n",
+    "  * [Set configurations](#Set-configurations)\n",
+    "  * [Create serializer and deserializer](#Create-serializer-and-deserializer)\n",
+    "  * [For visualization](#For-visualization)\n",
+    "* [Prepare data](#Prepare-data)\n",
+    "  * [Split a given dataset into train, validation, and test](#Split-a-given-dataset-into-train,-validation,-and-test)\n",
+    "* [Train multiple house value prediction models](#Train-multiple-house-value-prediction-models)\n",
+    "  * [Launch a single training job for a given housing location](#Launch-a-single-training-job-for-a-given-housing-location)\n",
+    "  * [Kick off a model training job for each housing location](#Kick-off-a-model-training-job-for-each-housing-location)\n",
+    "  * [Wait for all model training to finish](#Wait-for-all-model-training-to-finish)\n",
+    "  * [Move model artifacts to an S3 location](#Move-model-artifacts-to-an-S3-location)\n",
+    "* [Create Endpoint](#Create-Endpoint)\n",
+    "  * [Create model](#Create-model)\n",
+    "  * [Create endpoint config](#Create-endpoint-config)\n",
+    "  * [Create endpoint](#Create-endpoint)\n",
+    "* [Invoke endpoint](#Invoke-endpoint)\n",
+    "  * [Single record request: First Model](#Single-record-request:-First-Model)\n",
+    "  * [Single record request: Second Model](#Single-record-request:-Second-Model)\n",
+    "  * [Single record request, no explanation](#Single-record-request,-no-explanation)\n",
+    "  * [Batch request](#Batch-request)\n",
+    "* [Cleanup](#Cleanup)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ee0f8490",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "Amazon SageMaker Clarify helps improve your machine learning models by detecting potential bias and helping explain how these models make predictions. The fairness and explainability functionality provided by SageMaker Clarify takes a step towards enabling AWS customers to build trustworthy and understandable machine learning models.\n",
```

Comment:
I think we can improvement this paragraph to mention online explainability, and link to https://docs.aws.amazon.com/sagemaker/latest/dg/clarify-online-explainability.html .

Reason: The online explainability developer guide was not available when we built the first example notebooks, but now it is, so good to add cross-reference.

Or we can ship it first and have a new task to track the improvement to all of the online explainability example notebooks.

Speaking of cross-reference, it would be good to have a task tracking the doc updates. E.g. mention multi-model in the developer guide.

Reply:
> Or we can ship it first and have a new task to track the improvement to all of the online explainability example notebooks.

We already have a SIM for that: RAI-4083
For this notebook I will make this update right away

> it would be good to have a task tracking the doc updates. E.g. mention multi-model in the developer guide

Created new SIM for this: RAI-4228